### PR TITLE
added drop down for registries

### PR DESCRIPTION
**Pull request by msivasubramaniaan**
Fixes: https://github.com/redhat-developer/vscode-openshift-tools/issues/2978

**Post Fix:**
image

### CODE REVIEW

File: src/webview/devfile-registry/app/multipleSelect.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+const MenuProps = {
+  PaperProps: {
+    style: {
+      maxHeight: ITEM_HEIGHT * 4.5 + ITEM_PADDING_TOP,
+      width: 250,
+    },
+  },
+};
+
+const names = [
```
Comment (mohitsuman) [inline src/webview/devfile-registry/app/multipleSelect.tsx:26]: @msivasubramaniaan Why is this needed ? Remove these unnecessary code.

File: src/webview/devfile-registry/app/multipleSelect.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+import ListItemText from '@mui/material/ListItemText';
+import Select, { SelectChangeEvent } from '@mui/material/Select';
+import Checkbox from '@mui/material/Checkbox';
+
+const ITEM_HEIGHT = 48;
+const ITEM_PADDING_TOP = 8;
+const MenuProps = {
+  PaperProps: {
+    style: {
+      maxHeight: ITEM_HEIGHT * 4.5 + ITEM_PADDING_TOP,
```
Comment (mohitsuman) [inline src/webview/devfile-registry/app/multipleSelect.tsx:20]: @msivasubramaniaan It should auto fix the height, let's not hard code this.

File: src/webview/devfile-registry/app/multipleSelect.tsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+  return (
+    <div>
+      <FormControl sx={{ m: 1, width: 300 }}>
+        <InputLabel id='demo-multiple-checkbox-label'>All Devfiles</InputLabel>
+        <Select
+          labelId='demo-multiple-checkbox-label'
+          id='demo-multiple-checkbox'
+          multiple
+          value={personName}
```
Comment (mohitsuman) [inline src/webview/devfile-registry/app/multipleSelect.tsx:60]: Update this with devfileName.

**Review by mohitsuman — CHANGES_REQUESTED** [review]
Remove the unwanted code.

File: src/webview/devfile-registry/app/multipleSelect.tsx (hunk 3 above)
Comment (msivasubramaniaan) [reply src/webview/devfile-registry/app/multipleSelect.tsx]: This file was committed wrongly and deleted it nw

**Review by mohitsuman — APPROVED** [review]
lgtm
